Final_Project_Billinis.qmd

```diff
@@ -74,53 +71,49 @@ The `echo: false` option disables the printing of code (only output is displayed
 ```
 
 ```{r}
-install.packages("ggmap")
+#install.packages("ggmap")
 library(dplyr)
 library(tidyverse)
 library(lubridate)
 library(ggplot2)
-map_data(atlantic)
+library(basemaps)
+
 AerialAtl <- Grships %>% 
   filter(Event == "Aerial Bombing" & Location == "Atlantic")
 head(AerialAtl)
 ```
 
-
 ```{r}
 mapinfo <- map_data(world)
 ```
 
-
 ```{r}
 library(ggplot2)
 library(maps)
 library(tidyverse)
+library(sf) 
+library(rnaturalearth)
 mapdata <- map_data("world")
 view(mapdata)
 ```
 
-
 ```{r}
 mapdata <- left_join(mapdata, AerialAtl, by="region")
 ```
 
-
 ```{r}
-Atldata <- AerialAtl %>%
-  filter(ocean == "Atlantic")
-ggplot() +
-  geom_polygon(data = Altmap, aes(x = long, y = lat, group = group), fill = "purple", color = "white") +
-  geom_point(data = Atldata, aes(x = lon, y = lat), color = "lime green", size = 0.25) +
-  ggtitle("Aerial Bombardment Atlantic") +
-  theme_minimal()
+world <- ne_countries(scale = "medium", returnclass = "sf")
+Europe <- world[which(world$continent == "Europe"),]
+map <- ggplot(Europe) +
+  geom_sf() +
+  coord_sf(xlim = c(-25,50), ylim = c(35,70), expand = FALSE)
+map + geom_point(data=AerialAtl, mapping=aes(x=Long, y=Lat, group=NULL), color='blue')
```

Comment:
See https://www.rpubs.com/mkollontai/DATA607_TidyVerse for a good example of this.